### PR TITLE
implemented the namaka board stats to use data coming from the backen…

### PR DESCRIPTION
**Pull request by joshuaolusayo**
…d for the average rating

### CODE REVIEW

File: s/frontend/components/modals/views/report-user/report-user-modal-view.ts [hunk 1]
```diff
@@ -14,13 +14,14 @@ export function reportUserModalView({
 	modals.popup({
 		closeOnBlanketClick: true,
 		renderPopup: ({close}) => html`
-			<div class="modalview report">
+		<namaka-banned-user></namaka-banned-user>
```
Comment (chase-moskal) [inline s/frontend/components/modals/views/report-user/report-user-modal-view.ts:17]: let's revert changes to this file -- we don't actually want to put the "banned users" interface into the report-users modal view -- and this change is unrelated to board-stats anyways

File: s/frontend/components/namaka-board-stats/namaka-board-stats.ts [hunk 2]
```diff
@@ -2,19 +2,44 @@
...
+	
 	render() {
 		return html`
 			<div class="review-summary" part="container">
 				<!-- <namaka-my-review></namaka-my-review> -->
 				<div class="flex-container">
 					<div class="wrapper">
-						<div>
-							<span class="big">4.6</span>
+						{(this.stats?.scoring?.averageScore >= 0 && <div>
```
Comment (chase-moskal) [inline s/frontend/components/namaka-board-stats/namaka-board-stats.ts:37]: this is JSX syntax that doesn't work here

not working
```js
`<span>{value}</span>`
```

working
```js
`<span>${value}</span>`
```

**Review by chase-moskal — CHANGES_REQUESTED** [review]
we should have the board-stats component on the `index.html` page, so we can see that it works

File: s/frontend/components/modals/views/report-user/report-user-modal-view.ts (hunk 1 above)
Comment (joshuaolusayo) [reply s/frontend/components/modals/views/report-user/report-user-modal-view.ts]: Fixed

File: s/frontend/components/namaka-board-stats/namaka-board-stats.ts (hunk 2 above)
Comment (joshuaolusayo) [reply s/frontend/components/namaka-board-stats/namaka-board-stats.ts]: Fixed

**Review by chase-moskal — CHANGES_REQUESTED** [review]
we need this runtime error fixed

image
